packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -83,6 +83,19 @@ class InflationManager {
   }
 }
 
+const setGasParameters = async (
+  validatorUri: string,
+  validatorAddress: string,
+  gasCost: number
+) => {
+  const kit = newKit(validatorUri)
```

Comment:
Any reason not to use the default `kit` defined on line 197? Would allow to to avoid passing `validatorUri` and `validatorAddress` here and creating a new kit every time

Reply:
The `kit` from line 197 gets associated to different node and address at `startSyncNode`